pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>tech.stackable.trino</groupId>
     <artifactId>trino-opa-authorizer</artifactId>
-    <version>0.1.0-SNAPSHOT</version>
+    <version>377</version>
```

Comment:
I guess good decision as the interface seems to change quite often if i understand you correctly

Reply:
Yeah i normally do not like that but i think in this case we need to release / adapt to the trino changes alot. It maybe would make sense to still keep some actual product version like 0.1.0 to have some versioning.